drivers/sensor/Kconfig

```diff
@@ -75,6 +75,8 @@ source "drivers/sensor/lis2ds12/Kconfig"
 
 source "drivers/sensor/lis2mdl/Kconfig"
 
+source "drivers/sensor/lis2dw12/Kconfig"
```

Comment:
Move this up one line to preserve alpha order

Reply:
ack